File: app/src/main/java/com/example/sarada/moviereviews/models/ReviewResponse.java
package com.example.sarada.moviereviews.models;

import com.google.gson.annotations.SerializedName;

import java.util.List;

public class ReviewResponse {
    @SerializedName("id")
    private int id_review;
    @SerializedName("results")
    private List<Review> results;

    public int getId_review() {
        return id_review;
    }

    public void setId_review(int id_review) {
        this.id_review = id_review;
    }

    public List<Review> getResults() {
        return results;
    }

    public void setResults(List<Review> results) {
        this.results = results;
    }
}

File: app/src/main/java/com/example/sarada/moviereviews/models/ReviewResponse.kt
package com.example.sarada.moviereviews.models

import android.os.Parcelable
import com.google.gson.annotations.SerializedName
import com.example.sarada.moviereviews.models.Review
import com.squareup.moshi.Json
import kotlinx.android.parcel.Parcelize

@Parcelize
data class ReviewResponse(
    @Json(name = "id")
    var id_review: Int = 0,

    @Json(name = "results")
    var results: List<Review>? = null
):Parcelable